generators/app/templates/sources/main/_main.run.ts

```diff
@@ -84,6 +87,21 @@ function main($window: ng.IWindowService,
     updateTitle($state.current.data ? $state.current.data.title : null);
   });
 
+  /**
+   * Enables tracking by analytics service.
+   */
+  // HACK : ignore the first $viewContentLoaded event because it's actually fired once when uiView is instantiated,
+  // and then it's fired a second time after is has been linked. This is "by design" :-/
+  // (http://stackoverflow.com/questions/31000417/angular-js-viewcontentloaded-loading-twice-on-initial-homepage-load)
+  let loadedOnce = false;
+  vm.$on('$viewContentLoaded', function () {
```

Comment:
LOL, missed that since we are still on the 0.3.x version and the 1.x version has not been finalized yet 😅 

I have created a separate issue for the migration to using `$transitions` hooks (see #38 ), for the meanwhile it's still a valid change (until 1.0.0 version is out) that will get rid of the ugly hack, and it will simplify the migration task, as it will be needed anyway as we use it elsewhere.
Thanks for pointing that!